daemon/mgr/system.go

```diff
@@ -26,6 +26,12 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	unknownHostName      = "<unknown>"
```

Comment:
Is there is need to add three different const for "unknown", since they're all same ? Just looks weird

Reply:
Yeap, i have thought about this. Although their value are all same, they represent different value of system info field in unknown scene.

Reply:
I think it depends on how the three value diverged, it's possible to be different in the future?

Reply:
It may be hard to say.

Reply:
Yep, I think it's ok here.

Reply:
Yep, I think it's ok here.